### PR TITLE
Updated Dungeons section (Boss Collections)

### PR DESCRIPTION
**Pull request by MartinNemi03**
- Changed constants for boss collections
- Updated boss collections to count tier completions
- Added option to max a boss collection
- Also added Sadan as a Floor 6 boss

### CODE REVIEW

**Review by metalcupcake5 — APPROVED** [review]
~~martin stop spamming me kthx bai~~
Looks good

File: src/lib.js [hunk 1]
```diff
@@ -2387,31 +2391,62 @@ module.exports = {
...
+                let reward = coll.rewards[reward_id];
+                if (collections[floor_id].killed >= reward.required) 
+                    collections[floor_id].tier = reward.tier;
+                else break;
+
+                if (collections[floor_id].tier == coll.max_tiers)
+                    collections[floor_id].maxed = true;
+            }
+
+            console.log(collections[floor_id]);
```
Comment (boblovespi) [inline src/lib.js:2432]: please remove

**Review by boblovespi — CHANGES_REQUESTED** [review]
image
you need commas when number of kills > 1000

File: src/lib.js (hunk 1 above)
Comment (MartinNemi03) [reply src/lib.js]: cough, cough.. you've seen nothing

**Review by boblovespi — APPROVED** [review]

**Comment by metalcupcake5** [comment]
Martin you can delete the branch if you want

**Pull request merged**